overlord/ifacestate/ifacemgr.go

```diff
@@ -59,3 +114,43 @@ func (m *InterfaceManager) Ensure() error {
 func (m *InterfaceManager) Stop() error {
 	return nil
 }
+
+// Interfaces returns object holding a lists of all the plugs and slots and their connections.
+func (m *InterfaceManager) Interfaces() *interfaces.Interfaces {
```

Comment:
Yes but I, perhaps incorrectly, assumed that there's more to this than the repository. I think our parallel conversation will help me get the big picture and iterate on this branch. This is really meant to be a way to start the discussion more than production code.